images/varnish/Dockerfile

```diff
@@ -31,5 +30,8 @@ EXPOSE 8080
 # tells the local development environment on which port we are running
 ENV LAGOON_LOCALDEV_HTTP_PORT=8080
 
+# Make http_resp_hdr_len and http_resp_size configurable
+ENV HTTP_RESP_HDR_LEN=${http_resp_hdr_len:-8k}
```

Comment:
this will not allow to overwrite the variables in runtime, but instead in Buildtime which I think it would be good to happen in Runtime

Reply:
or actually it does allow to overwrite the variables, but then we don't need the `${http_resp_hdr_len:-8k}` in there, and just use `HTTP_RESP_HDR_LEN=8k`

Reply:
updated :)